test/e2e/servers/ory/kratos.spec.ts

```diff
@@ -432,40 +390,88 @@ describe("phone+email schema", () => {
     const newIdentity = await kratosAdmin.getIdentity({ id: kratosUserId })
     expect(newIdentity.data.schema_id).toBe("phone_email_no_password_v0")
     expect(newIdentity.data.traits.email).toBe(email)
-    expect(newIdentity.data.verifiable_addresses?.[0].verified).toBe(false)
+
+    expect(await authServiceEmail.isEmailVerified({ email })).toBe(false)
+  })
+
+  it("can't add same email to multiple identities", async () => {
+    const phone = randomPhone()
+    const res0 = await authServicePhone.createIdentityWithSession({ phone })
+    if (res0 instanceof Error) throw res0
+    const kratosUserId = res0.kratosUserId
+
+    const res = await authServiceEmail.addUnverifiedEmailToIdentity({
+      kratosUserId,
+      email,
+    })
+    if (!(res instanceof AuthenticationError)) throw new Error("wrong type")
+    expect(res.message).toBe("Request failed with status code 409")
+  })
+
+  it("can't authenticate until email has been verified", async () => {
+    // this check should be at a higher level
+    expect(true).toBe(true)
   })
```

Comment:
could somebody add an email address that does not belong to them and never verify it. Then when the rightful owner tries to add it, would they be blocked?

Reply:
I think currently we can only add one email with the admin api we are using. it will need to be deleted before attempting to add another one (for a typo for instance). 

this is because the email is attached to the identity before it has been verified. it's a flaw somewhat but I'm not sure there is an alternative in how we use the API.

so in effect yes it could be an issue, but an "email spammer" would be able to add only one email. 

I've been thinking about a way to clean this up eventually, with a cronjob that would remove unconfirmed email every day or so. maybe not very important at first.

Reply:
Good idea, a cron job would be a nice simple solution. I think the email verification code expires after 1 hour by default so a daily cron should be good enough to mitigate